src/main/java/de/qabel/desktop/ui/contact/ContactController.java

```diff
@@ -247,6 +255,15 @@ protected void buildGson() throws EntityNotFoundExcepion, PersistenceException {
 		builder.registerTypeAdapter(Contact.class, new ContactTypeAdapter());
 		gson = builder.create();
 	}
+
+	@Override
+	public void update() {
+		try {
+			loadContacts();
```

Comment:
```JAVA
Platform.runLater(...);
```
you don't know if the thread that removed the contact (or added one) is the PlaformThread